commcare_connect/reports/helpers.py

```diff
@@ -169,5 +176,23 @@ def get_table_data_for_year_month(
         top_five_percent_len = len(sum_total_users) // 20 or 1
         flw_amount_paid = sum(sum_total_users.values())
         avg_top_paid_flws = sum(sorted(sum_total_users.values(), reverse=True)[:top_five_percent_len])
-        visit_data_dict[group_key].update({"flw_amount_paid": flw_amount_paid, "avg_top_paid_flws": avg_top_paid_flws})
+        visit_data_dict[group_key].update(
+            {
+                "month_group": datetime.strptime(month_group, "%Y-%m"),
+                "delivery_type_name": delivery_type_name,
+                "flw_amount_paid": flw_amount_paid,
+                "avg_top_paid_flws": avg_top_paid_flws,
+            }
+        )
+
+    connectid_user_count = fetch_user_counts()
+    total_connectid_user_count = 0
+    total_eligible_user_count = 0
+    for group_key in visit_data_dict.keys():
```

Comment:
If it is broken out by delivery type, what value is in each row? The same for a given month, or something else?